src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -634,4 +634,7 @@
   <data name="WindowRestoreDownButtonToolTip" xml:space="preserve">
     <value>Restore Down</value>
   </data>
+  <data name="CommandPaletteMenuItem" xml:space="preserve">

```

Comment:
Should we remove "FeedbackMenuItem" if no one is using it anymore?

Reply:
no-can-do!

![image](https://user-images.githubusercontent.com/189190/121573015-5e816280-c9ea-11eb-90f1-6493915ed8e6.png)

Reply:
The localizations are not versioned (and there does not appear to be a capacity _for_ versioning them!)

Reply:
WELP NEVERMIND THEN.